Testscripts/Windows/VALIDATE-INTEL-SGX-DRIVER.ps1

```diff
@@ -20,6 +20,17 @@ function Main {
 
     $ubuntuVersion = Run-LinuxCmd -Command "cat /etc/issue" `
         -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort
+
+    if (($ubuntuVersion -imatch "Ubuntu 18.04") -or ($ubuntuVersion -imatch "Ubuntu 16.04") -or ($ubuntuVersion -imatch "Ubuntu 20.04") -or ($ubuntuVersion -imatch "Ubuntu 19.10")) {
+        $retVal = Run-LinuxCmd -Command "lsmod | grep -i intel_sgx" -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort -ignoreLinuxExitCode
+        if (!$retVal) {
```

Comment:
what if the mistaken module name is 'intel_sgxx'? It will pass, right?

Reply:
it will fail in the bash script if intel_sgx not load, which module named intel_sgxx?

Reply:
acked